app/services/finance/npq/statement_calculator.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    class StatementCalculator
+      attr_reader :statement
+
+      def initialize(statement:)
+        @statement = statement
+      end
+
+      def summary_overall_total
+        total_overview_payment + overall_vat
+      end
+
+      def total_output_payment_subtotal
+        output_payment.sum { |output_payment| output_payment[:subtotal] }
+      end
+
+      def total_service_fees
+        service_fees.sum { |service_fee| service_fee[:monthly] }
+      end
+
+      def overall_vat
+        total_payment * (npq_lead_provider.vat_chargeable ? 0.2 : 0.0)
+      end
+
+      def total_payment
+        total_service_fees + total_output_payment_subtotal
+      end
+
+      def total_starts
+        statement_declarations.where(declaration_type: "started").count
+      end
+
+      def total_retained
+        statement_declarations.where(declaration_type: %w[retained-1 retained-2]).count
```

Comment:
```suggestion
        statement_declarations.retained_1.retained_2.count
```

Reply:
I've added new scope `retained` to deal with them all